tests/drivers/gpio/gpio_arduino/testcase.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  tags:
+    - drivers
+    - gpio
+  depends_on:
+    - gpio
+    - arduino_gpio
```

Comment:
are both features required? I see boards which supports only arduino_gpio but not gpio (not sure it it is intended) e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/mimxrt1010_evk/mimxrt1010_evk.yaml

Reply:
this is the board yaml issue, gpio is a must. let me create another PR to fix

Reply:
pr created #70202 70202